common/static/common/js/components/views/tabbed_view.js

```diff
@@ -23,9 +34,13 @@
                    initialize: function (options) {
                        this.url = options.url;
                        this.view = options.view;
+                       this.index = options.index;
                    },
-                   render: function () {
-                       var tabPanelHtml = this.template({tabId: getTabPanelId(this.url)});
+                   render: function() {
+                       var tabPanelHtml = this.template({
+                               tabId: getTabPanelId(this.url),
+                               index: this.index
```

Comment:
What does ``index`` mean? Is this the selected index? The index of tabbed components on the page?